packages/jest-config/index.js

```diff
@@ -1,8 +1,12 @@
+/**
+ * @type {import('@jest/types').Config.InitialOptions}
+ */
 module.exports = {
   testEnvironment: "node",
   testMatch: ["<rootDir>/src/**/*.test.ts"],
   transform: {
     "^.+\\.tsx?$": "esbuild-jest",
+    "^.+\\.webp$": require.resolve("./file-transform.js"),
```

Comment:
alternatively, you could use a "reverse" regex like `"^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)"`. This would handle every non-js/css/json asset with this `file-transform`. I'm personally more comfortable with the explicit allowlist though.